providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/FlagdProviderTest.java

```diff
@@ -26,6 +25,7 @@
 import dev.openfeature.sdk.Reason;
 import dev.openfeature.sdk.Structure;
 import dev.openfeature.sdk.Value;
+import io.cucumber.java.AfterAll;
```

Comment:
Changes in this file are not related, just doing some clenaup of unused vars and reducing log noise.